templates/rhdm70-kieserver.yaml

```diff
@@ -90,7 +90,8 @@ parameters:
   name: KIE_SERVER_CONTROLLER_PROTOCOL
   value: http
   required: false
-- displayName: KIE server controller service 
+- displayName: KIE server controller service
+  description: The service name for the optional business-central-monitor, where it can be reached and registered with, to allow monitoring console functionality
```

Comment:
@spolti Shouldn't this reference decision-central (for RHDM) vs. business-central-monitoring (for RHPAM)?